From ce078adfa63ecfeac4638fcb1f3f3a16b68ac7a4 Mon Sep 17 00:00:00 2001
From: Mathias Agopian <mathias@google.com>
Date: Fri, 17 Feb 2017 18:40:46 -0800
Subject: [PATCH 1/3] fix fugu build attempt. fallout of splitting rect.h out
 of libandroid.
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Some makefiles didn’t specify libgui but included
headers from it directly, and they didn’t even
need them.

It works because these headers are on the global
include path. With this change, though, rect.h
is not anymore (albeit exported from libui).

Test: built
Bug: 35164655
Change-Id: I72da3f159db289bf1c1466b095369f19a38ff923
---
 src/android/psb_android_glue.cpp | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/src/android/psb_android_glue.cpp b/src/android/psb_android_glue.cpp
index 0ac3f27..122c2f8 100755
--- a/src/android/psb_android_glue.cpp
+++ b/src/android/psb_android_glue.cpp
@@ -31,14 +31,10 @@
 #include <binder/IPCThreadState.h>
 #include <binder/ProcessState.h>
 #include <binder/IServiceManager.h>
-#include <gui/Surface.h>
-#include <gui/SurfaceComposerClient.h>
-#include <gui/ISurfaceComposer.h>
 #include <binder/MemoryHeapBase.h>
 #include "psb_android_glue.h"
 #include "psb_output_android.h"
 #include <cutils/log.h>
-#include <ui/Rect.h>
 #include <system/window.h>
 #include <system/graphics.h>
 #ifdef TARGET_HAS_MULTIPLE_DISPLAY
-- 
2.20.1

